Server/mods/deathmatch/logic/CResource.cpp

```diff
@@ -745,9 +745,13 @@ bool CResource::Start(std::list<CResource*>* pDependents, bool bManualStart, con
     if (!GetCompatibilityStatus(strStatus))
     {
         m_strFailureReason = SString("Not starting resource %s as %s\n", m_strResourceName.c_str(), strStatus.c_str());
+#ifdef MTA_DEBUG
```

Comment:
How about just using the build_overrides.h to define when you want to skip version checks, much like here: https://github.com/multitheftauto/mtasa-blue/blob/a0410c68945b6b3d9225c8c90af2bb6b1cda5eaa/Server/core/CDynamicLibrary.cpp#L148